tsconfig.json

```diff
@@ -18,7 +18,16 @@
   "typedocOptions": {
     "mode": "file",
     "module": "commonjs",
+    "excludeExternals": true,
     "excludePrivate": true,
-    "excludeNotExported": true
+    "excludeNotExported": true,
+    "name": "Unstoppabledomains resolution library",
+    "inputFiles": [
+      "./src/Resolution.ts",
+      "./src/index.ts",
```

Comment:
@JohnnyJumper do we need index?

Reply:
this index is everything that we are exposing to the end-user. They are able to import and use all of what is exported from that file